diff --git a/minio_backend/minio_backend.py b/minio_backend/minio_backend.py
index 158001d..87aa2a0 100644
--- a/minio_backend/minio_backend.py
+++ b/minio_backend/minio_backend.py
@@ -3,6 +3,7 @@ import logging
 logger = logging.getLogger('openerp.' + __name__)
 
 
+
 def get_minio_client():
     from minio import Minio
     from tools.config import config
@@ -14,7 +15,9 @@ def get_minio_client():
         raise ValueError('Missing Minio Server configuration')
     if not secure:
         logger.warning('Using an insecure connection to Minio server.')
+    import urllib3
+    httpClient = urllib3.PoolManager(cert_reqs='CERT_REQUIRED', ca_certs='/home/erp/conf/minio_cert/public.crt')
     client = Minio(
-        endpoint, access_key=access_key, secret_key=secret_key, secure=secure
+        endpoint, access_key=access_key, secret_key=secret_key, secure=secure, http_client=httpClient
     )
     return client
